.github/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+# Pull Request

```

Comment:
remove this line

Reply:
Fixed in [f21530d](https://github.com/AIHawk-FOSS/Auto_Jobs_Applier_AI_Agent/pull/910/commits/f21530dc055f8fa5c1f1d449c814654489bc150a)